release-notes-4.1.0.md

```diff
@@ -1,8 +1,14 @@
-!not-ready-for-release!
 
 #### Version Number
 ${version-number}
 
 #### New Features
+ - **SCMOD-10189**:  Updated Tomcat base image.  
+  This has changes to meet the logging mandate by adding caf logging Tomcat juli project to Tomcat Java 11 image. 
+
+#### Bug Fixes
+ - **SCMOD-9495**: Adding the REQUIRED_DOCKER_HOST_TYPE=PRIVILEGED setting to the official-build.props file.  
```

Comment:
Drop this.  This is not fixing a bug that was present in a previously released version.

Reply:
you mean SCMOD-9495 ?

Reply:
Sorry didn't get what you are saying?

Reply:
Sorry didn't get what you are saying?

Reply:
Yes that shouldn't be mentioned in any release notes as it is a change to how the product is built as opposed to being a change to the product.  Think of it like this - downstream consumers are totally unaffected by it.

Reply:
ok got it